objects/src/accounts/storage/map.rs

```diff
@@ -85,12 +98,29 @@ impl StorageMap {
         self.map.inner_nodes() // Delegate to Smt's inner_nodes method
     }
 
-    // STATE MUTATORS
+    // DATA MUTATORS
     // --------------------------------------------------------------------------------------------
-
     pub fn insert(&mut self, key: RpoDigest, value: Word) -> Word {
         self.map.insert(key, value) // Delegate to Smt's insert method
     }
+
+    /// Applies the provided delta to this account storage.
+    ///
+    /// This method assumes that the delta has been validated by the calling method and so, no
+    /// additional validation of delta is performed.
+    pub(super) fn apply_delta(&mut self, map_delta: StorageMapDelta) -> Result<(), AccountError> {
```

Comment:
I think we could pass the `delta` parameter by reference (and we can call it just `delta`). Also, I think it is OK to make this function fully public. For example:
```Rust
pub fn apply_delta(&mut self, delta: &StorageMapDelta) -> Result<(), AccountError> {
```

Reply:
done